README.md

```diff
@@ -107,12 +111,15 @@ public class TestActivity extends AppCompatActivity {
     @Override
     protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
-        Belvedere.from(this).getFilesFromActivityOnResult(requestCode, resultCode, data, new Callback<List<MediaResult>>() {
+        
+        callback = new Callback<List<MediaResult>> () {
             @Override
             public void success(List<MediaResult> result) {
-                // Handle Selected files
+                // Handle selected files                
             }
-        });
+        };
+        
+        Belvedere.from(this).getFilesFromActivityOnResult(requestCode, resultCode, data, callback);
```

Comment:
This usage will currently cause an activity leak. 
Can u update to show that the application context should be used or change our internals to operate on the application context instead?

Reply:
`Belvedere.from(ctx)` get the application context. 

See:
https://github.com/zendesk/belvedere/blob/master/belvedere-core/src/main/java/zendesk/belvedere/Belvedere.java#L58

Reply:
It's still possible to construct Belvedere instance that operates on any context, see https://github.com/zendesk/belvedere/blob/master/belvedere-core/src/main/java/zendesk/belvedere/Belvedere.java#L286

Reply:
Fair enough., but that would not be the case in this very example, right?

Reply:
Could be if https://github.com/zendesk/belvedere/blob/master/belvedere-core/src/main/java/zendesk/belvedere/Belvedere.java#L75 was called prior to the example code

Reply:
If `setSingletonInstance` before this call the context passed in here would not be used at all. 

I'd agree to put a `getApplicationContext` into the builder as a safety mechanism, but for simplicity and cause there's no harm keep`from(this)` as it is.

Reply:
Added in latest commit.